web/models/job.ex

```diff
@@ -48,7 +49,7 @@ defmodule Community.Job do
   def admin_changeset(model, params \\ %{}) do
     model
     |> cast(params, @required_fields, @optional_fields)
-    |> validate_format(:company_url, ~r/http(s)?.*/, message: "must start with http(s)")
+    |> validate_url_format(:company_url)
```

Comment:
In case like this I think it is easier to use the fully qualified version of an alias. Otherwise it looks like it's part of Ecto. Thoughts?